test/regression.uts

```diff
@@ -1568,6 +1568,23 @@ beacon = frame.getlayer(5)
 ssid = beacon.network_stats()['ssid']
 assert ssid == "ROUTE-821E295"
 
+= SSID is parsed properly even when the Country Information Tag Element has an odd length (not complying with the standard) and a missing pad byte
+~ Missing Pad Byte in Country Info
+# A regression test for https://github.com/secdev/scapy/pull/2685.
+# https://github.com/secdev/scapy/issues/4132 describes a packet with
+# a Country Information element tag that has an odd length, even though it's against the standard.
+# The transmitter should have added a padding byte to make the length even, but it didn't.
+# The effect on scapy used to be improper parsing of the next tag elements, causing the SSID to be overridden.
+# This test checks the SSID is parsed properly.
+filename = scapy_path("/test/pcaps/dot11_country_info_missing_pad_byte.pcap")
```

Comment:
For such a small packet there's no need to put a pcap, you should embed the packet in the test as bytes.